src/Uno.UI/UI/Xaml/Style/Generic/SystemResources.xaml

```diff
@@ -631,6 +631,8 @@
 			<SolidColorBrush x:Key="HyperlinkPointerOverForegroundThemeBrush" Color="#CC9C72FF" />
 			<SolidColorBrush x:Key="HyperlinkPressedForegroundThemeBrush" Color="#999C72FF" />
 			<StaticResource x:Key="HyperlinkForeground" ResourceKey="SystemControlHyperlinkTextBrush" />
+			<StaticResource x:Key="HyperlinkForegroundPointerOver" ResourceKey="SystemControlPageTextBaseMediumBrush" />
```

Comment:
Should this be `SystemControlHyperlinkBaseMediumBrush`?

Reply:
Yes. My bad.